src/server/game/Entities/Object/Object.cpp

```diff
@@ -2566,7 +2566,8 @@ void WorldObject::GetContactPoint(const WorldObject* obj, float& x, float& y, fl
     // angle to face `obj` to `this` using distance includes size of `obj`
     GetNearPoint(obj, x, y, z, obj->GetObjectSize(), distance2d, GetAngle(obj));
 
-    if (fabs(this->GetPositionZ() - z) > 3.0f || !IsWithinLOS(x, y, z))
+    // Exclude gameobjects from LoS calculations
+    if (fabs(this->GetPositionZ() - z) > 3.0f || (GetTypeId() != TYPEID_GAMEOBJECT && !IsWithinLOS(x, y, z)))
```

Comment:
I'm sorry if it's a dumb question, but this won't affect Sapphiron/Sindragosa's tombs, right?

Reply:
It won't.